docs/docs/start-here.md

```diff
@@ -56,35 +55,36 @@ then [send us a mail](mailto:jspecify-discuss@googlegroups.com)
 * You might wish to be an early adopter of **version 0.3** in your projects;
   see the [adoption](https://github.com/jspecify/jspecify/wiki/adoption) wiki
   page for advice.
-* When our **reference implementation** is ready (targeting 2023 Q1), you'll be
-  able to validate your usages of the annotations against our defined semantics.
-  This is when you will really get to find out how helpful or annoying our
-  current design choices are for you, and we'll be glad to hear about it.  This
-  checker is not quite ready for prime time yet, however. It can be found, but
-  we're not linking to it yet.
+* Please experiment with our 
+  **[reference implementation](https://github.com/jspecify/jspecify-reference-checker)**.
+  This lets you validate your usages of the annotations against our defined
+  semantics, which is when you will really get to find out how helpful or 
+  annoying our current design choices are for you (which you should let us
+  know!). However, this tool is still a work in progress, and is not at *full*
+  conformance with our own specification quite yet.
 * **Version 1.0** will commit **permanently** to the annotation APIs, and they
-  will be safe to depend on. From that point, no incompatible changes from a
-  compilation perspective will happen, and any further specification adjustments
-  will not break null-checking for you or your clients except to improve rare
-  corner cases. We have no prediction for when this release will be. We hope to
-  get as much of your feedback as we can before then.
+  will be safe to depend on. From that point, no compile-breaking changes will
```

Comment:
minor and preexisting point:

I wonder if users will in fact understand "compile-breaking" to mean "no breakages from the stock javac" as opposed to "no breakages produced by nullness checkers in Java or Kotlin clients."

Is it worth saying something like "We will maintain source and binary compatibility" instead?

Reply:
I've spent some hems and haws over this. I think I'm gonna leave it as a future improvement for now, thanks for approving. :-)